src/components/movieInfo/movieInfo.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { hot } from "react-hot-loader";
+import "./movieInfo.scss";
+import PropTypes from "prop-types";
+
+
+const MovieInfo = (props) => (
+    <article className="movieInfo">
+        <div className="movieInfo__imgWrapper">
```

Comment:
Looks like a shared component could be created for poster (as you use it in movieCard too).

Reply:
Created common component